### PR TITLE
50507 Accessibility changes for all warning pages

### PR DESCRIPTION
**Pull request by ganpat-tech**
PR contain changes in JS , cshtml and CSS file.

### CODE REVIEW

File: UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/wwwroot/js/Custom/Common.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
+﻿document.addEventListener("DOMContentLoaded", function () {
+    HashLinkTab();
```
Comment (usha13529) [inline UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/wwwroot/js/Custom/Common.js:2]: As mentioned above, please move all relevant changes to site.js and remove common.js file.

**Review by usha13529 — CHANGES_REQUESTED** [review]
Two review comments to look into please

File: UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/Shared/_MSILayout.cshtml [hunk 2]
```diff
@@ -24,6 +24,8 @@
     </div>
 </div>
 
+<script type="text/javascript" src="@Url.Content("~/js/Custom/Common.js")"></script>
```
[the remark this answers is not in the document]
Comment (mubeenMastek) [reply UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/Shared/_MSILayout.cshtml]: Removed this line of code

**Review by usha13529 — APPROVED** [review]
Thanks @mubeenMastek for changes. Looks good to me.

**Review by barrie-cooper — APPROVED** [review]
LGTM, thanks @ganpat-tech & @mubeenMastek

**Pull request merged**